src/keyboard.c

```diff
@@ -14,6 +14,8 @@ See the file LICENSE for details.
 
 #define KEY_INVALID 0127
 
+#define W_SUBSTITUTE -1
```

Comment:
I see that, and that's why I proposed 255 (W is apparently a type-able char, can we type ASCII 255 from keyboard?) If there's no better alternative, I'm happy to merge this.